contractcourt/htlc_incoming_contest_resolver.go

```diff
@@ -114,6 +114,11 @@ func (h *htlcIncomingContestResolver) Resolve() (ContractResolver, error) {
 		blockEpochs.Cancel()
 	}()
 
+	hodlSubscription := h.HodlManager.Subscribe()
+	defer hodlSubscription.Cancel()
+
+	hodlSubscription.Subscribe(h.payHash)
```

Comment:
No error or feedback needed here?

Reply:
Call has been removed and integrated with `NotifyExitHopHtlc`.